src/components/layout/yearInReview/2021/communityGrowth.tsx

```diff
@@ -53,12 +71,22 @@ const growthPoints: {
 const imageDimension = 150;
 
 const CommunityGrowth: React.FC = () => {
+  const intl = useIntl();
   return (
     <>
       <SectionContainer
         header={
           <SectionHeader
-            header={['Encouraging', 'community growth']}
+            header={[
```

Comment:
Please leave this out for now, I'll look for a solution to translate this in one message.